composer.json

```
@@ -53,7 +53,7 @@
         "phpstan/phpstan-symfony": "^1.0",
         "psr/log": "^1.0 || ^2.0 || ^3.0",
         "ramsey/uuid": "^3.7 || ^4.0",
```

```suggestion
        "ramsey/uuid": "^3.9.7 || ^4.0",
```

See @ https://github.com/ramsey/uuid/issues/439

Also `symfony/dependency-injection` needs to be pinned to ^6.1.12 due to bug @ https://github.com/symfony/symfony/pull/49126

Thanks!

---

Thanks, updated!